### PR TITLE
Clean up redundant ones and do corrections

### PR DESCRIPTION
**Pull request by viethoangpyco**

### CODE REVIEW

File: nuget.config [hunk 1]
```diff
@@ -5,7 +5,6 @@
   -->
     <packageSources>
         <clear />
-        <add key="LocalTest" value="c:\projects\sitecore.habitathome.commerce\lib\LocalRepository" />
```
Comment (viethoangpyco) [inline nuget.config]: @jeanfrancoislarente The local source 'c:\projects\sitecore.habitathome.commerce\lib\LocalRepository' doesn't exist.

File: deploy-commerce-engine.ps1 [hunk 2]
```diff
@@ -85,8 +85,8 @@ Function  Start-CommerceEnginePepare ( [string] $basePublishPath = $(Join-Path $
     $idServerSettings | ConvertTo-Json -Depth 10 -Compress | set-content $idServerJson
 
     #Modify BizFx to match new hostname
-    $bizFxJson = $([System.IO.Path]::Combine($webRoot, $BizFxPathName, "assets\config.json")
-    $bizFxSettings = $bizFxJson | Get-Content $pathToJson -Raw | ConvertFrom-Json)
+    $bizFxJson = $([System.IO.Path]::Combine($webRoot, $BizFxPathName, "assets\config.json"))
+    $bizFxSettings = Get-Content $bizFxJson -Raw | ConvertFrom-Json
```
Comment (viethoangpyco) [inline deploy-commerce-engine.ps1]: @jeanfrancoislarente all updates in this file are for solving the following issues
hh902_5

File: HabitatHome.Commerce.sln [hunk 3]
```diff
@@ -1,299 +1,301 @@
...
+		.gitattributes = .gitattributes
+		.gitignore = .gitignore
+		deploy-commerce-engine.ps1 = deploy-commerce-engine.ps1
+		gulp-config.js = gulp-config.js
+		gulpfile.js = gulpfile.js
+		nuget.config = nuget.config
+		package.json = package.json
+		publishsettings.targets = publishsettings.targets
+		README.md = README.md
+		src\Project\HabitatHome\website\App_Config\Include\Project\z.HabitatHome.Commerce.Website.DevSettings.config = src\Project\HabitatHome\website\App_Config\Include\Project\z.HabitatHome.Commerce.Website.DevSettings.config
```
Comment (viethoangpyco) [inline HabitatHome.Commerce.sln]: @jeanfrancoislarente add this one to Configuration folder

File: HabitatHome.Commerce.sln [hunk 4]
```diff
@@ -1,299 +1,301 @@
...
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Configuration", "Configuration", "{38E46594-2B99-43FF-9E92-9A11B45168D8}"
+	ProjectSection(SolutionItems) = preProject
+		.gitattributes = .gitattributes
+		.gitignore = .gitignore
+		deploy-commerce-engine.ps1 = deploy-commerce-engine.ps1
+		gulp-config.js = gulp-config.js
+		gulpfile.js = gulpfile.js
+		nuget.config = nuget.config
+		package.json = package.json
+		publishsettings.targets = publishsettings.targets
```
Comment (viethoangpyco) [inline HabitatHome.Commerce.sln]: @jeanfrancoislarente add this one to Configuration folder

**Comment by akshaysura** [comment]
Great work @viethoangpyco that resolved my issues.

**Pull request merged**